scss/bitstyles/ui/common-conventions.stories.mdx

```diff
@@ -0,0 +1,16 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="Common conventions" />
+
+# Common conventions
+
+## Classname prefixes
+
+`.u-`, `.a-` etc
+
+## Breakpoint suffixes
+
+`@m`, `@xl` etc
+
+## centralised design tokens & helper functions
```

Comment:
```suggestion
## Centralised design tokens & helper functions
```

Reply:
I rewrote this section in a later commit 👍